comp/core/flare/helpers/send_flare.go

```diff
@@ -122,6 +122,7 @@ func readAndPostFlareFile(archivePath, caseID, email, hostname, url string, sour
 	if err != nil {
 		return nil, err
 	}
+	request.Header.Add("Authorization", fmt.Sprintf("Bearer %s", apiKey))
```

Comment:
```suggestion
	request.Header.Add("DD-API-KEY", apiKey)
```
Otherwise it wont work

Reply:
fixed in https://github.com/DataDog/datadog-agent/pull/27912/commits/cef82e5796c2396bc3059de19db11f0dfd68b2bc